lib/pf/violation.pm

```diff
@@ -894,15 +887,17 @@ sub violation_maintenance {
     my $rows_processed = 0;
     while(1) {
         my ($status, $iter) = pf::dal::violation->search(
-            {
-                -where => {
-                    status => ["open", "delayed"],
-                    release_date => [-and => {"!=" => 0}, {"<=" => \'NOW()'}],
-                },
-               -limit => $batch,
-               -columns => [qw(id mac vid notes status)],
-            }
+            -where => {
+                status => ["open", "delayed"],
+                release_date => [-and => {"!=" => $ZERO_DATE}, {"<=" => \'NOW()'}],
+            },
+           -limit => $batch,
+           -columns => [qw(id mac vid notes status)],
+           -from => pf::dal::violation->table,
```

Comment:
useful ?

Reply:
Yes it is I did not want to hard code the table name

Reply:
What I mean is that shouldn't this already be the case when using the violation DAL object ?

Reply:
As jazzed verbally, if joins should be skipped, it should be done via an explicit parameter instead of this workaround so its more explicit to the untrained eye